bots/naughty/debian-stable/5399-storaged-crash-poll-with-variant

```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
+  File "./verify/check-storage-lvm2", line 155, in testLvm
```

Comment:
I just copied them from debian-testing. However, it seems that #5399 only happened once, in March, on debian-testing. And #5673 has basically never matched. Also, the tests succeed in our current logs, so I think we can just drop these files altogether.